rest/taskana-rest-spring/src/main/java/pro/taskana/rest/WorkbasketDefinitionController.java

```diff
@@ -129,25 +129,26 @@
     // STEP 1: update or create workbaskets from the import
     for (WorkbasketDefinitionResource definition : definitions) {
       Workbasket importedWb = workbasketDefinitionAssembler.toModel(definition.getWorkbasket());
-      Workbasket workbasket;
+      String newId;
+      Workbasket wbWithoutId = removeId(importedWb);
       if (systemIds.containsKey(logicalId(importedWb))) {
-        workbasket = workbasketService.updateWorkbasket(importedWb);
+        workbasketService.updateWorkbasket(wbWithoutId);
+        newId = systemIds.get(logicalId(importedWb));
```

Comment:
I changed this, because the returned workbasket could contain a wrong id.
This happens in the case of an workbasket with existing key-domain-value but wrong id.
In this case the id should also be removed because the backend doesn't check this properly.